### PR TITLE
Clear docker host test + support port

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

**Comment by meirwah** [comment]
@orius123 please review...

**Comment by meirwah** [comment]
Please note that i'm doing "Docker rm" cmd as part of the flow...

File: test/io/cloudslang/docker/maintenance/test_clear_docker_host.sl [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+    - run_container:
+        do:
+          containers.create_container:
+            - host
+            - port:
+                required: false
+            - username
+            - password
+            - containerName: "'xxx'"
```
Comment (orius123) [inline test/io/cloudslang/docker/maintenance/test_clear_docker_host.sl]: ?

**Comment by orius123** [comment]
how does it pass on circle ci??

**Comment by meirwah** [comment]
good question...

File: test/io/cloudslang/docker/maintenance/test_clear_docker_host.sl (hunk 1 above)
Comment (meirwah) [inline test/io/cloudslang/docker/maintenance/test_clear_docker_host.sl]: containerName is not optional right now..
we need to refactor it..

Comment (meirwah) [inline test/io/cloudslang/docker/maintenance/test_clear_docker_host.sl]: just gave it an arbitrary name

**Pull request merged**